src/Microsoft.HttpRepl.IntegrationTests/Commands/ClearCommandTests.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.IntegrationTests.Mocks;
+using Microsoft.Repl;
+using Microsoft.Repl.Commanding;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Commands
+{
+    public class ClearCommandTests : CommandTestsBase
+    {
+        [Fact]
+        public void CanHandle_Name_ReturnsTrue()
+        {
+            ArrangeInputs("clear", out MockedShellState shellState, out HttpState httpState, out ICoreParseResult parseResult);
```

Comment:
nit: can we specify the parameter here for readability?

Reply:
Fixed in https://github.com/aspnet/HttpRepl/pull/74/commits/af55ecc8092914f16ced3d5fd83db4b96b4de1de